tests/playwright/src/model/pages/extensions-page.ts

```diff
@@ -108,4 +108,10 @@ export class ExtensionsPage {
     const extension = this.content.getByRole('region', { name: label });
     return (await extension.count()) > 0;
   }
+
+  public async getInstalledExtensionVersion(name: string, label: string): Promise<string> {
+    const extensionCard = await this.getInstalledExtension(name, label);
+    const version = extensionCard.rightActions.getByLabel('Version');
+    return (await version.textContent()) ?? '';
```

Comment:
You should change the return value to `Promise<string | undefined>`.
Then you should check if the element count is greater than 0 and then return `innerText` if it is non-empty, `undefined` otherwise.

Or you can simply define the locator in ExtensionCard and assert using `toHaveText`. etc.